digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/RoadwayChangesDAO.scala

```diff
@@ -593,8 +593,12 @@ SELECT
                    |	new_end_addr_m - new_start_addr_m,
                    |	new_administrative_class
                    |FROM roadway_changes rc
-                   |JOIN road_name rn ON rc.new_road_number = rn.road_number AND rn.end_date IS NULL AND rn.valid_to IS NULL
-                   |JOIN project p ON rc.project_id = p.id""".stripMargin
+                   |JOIN project p ON rc.project_id = p.id
+                   |JOIN road_name rn ON (rc.new_road_number = rn.road_number OR  rc.old_road_number = rn.road_number)
+                   |  AND rn.valid_to IS NULL
```

Comment:
... ja tosta tuli mieleen, päteekö toi "rn.valid_to IS NULL" myös lakkautetuille teille, vai tuleeko tyhjää?

Reply:
valid_to != null on debug rivi myös road name taulussa -> sillä ei tehdä siis mitään -> suodatetaan pois kyselyssä